src/pages/levels/Controls.js

```diff
@@ -0,0 +1,113 @@
+import { useNavigate } from '@reach/router';
+import {
+  FormControl,
+  FormControlLabel,
+  Grid,
+  InputLabel,
+  MenuItem,
+  Select,
+  Switch,
+} from '@material-ui/core';
+import React from 'react';
+import Popularity from 'components/Popularity';
+import styled from 'styled-components';
+
+const Controls = ({ detailed, sort }) => {
+  const navigate = useNavigate();
+
+  return (
+    <Grid2
+      container
+      justify="flex-end"
+      alignItems="center"
+      className="controls"
```

Comment:
classNames shouldn't really be necessary with styled components, and don't see it used anywhere

Reply:
class names make it much easier to modify from the parent of the component that uses it or anything above that. i'm not so sure if styled components offer any good solutions for stuff like that. In this case I don't need the class name though so I have removed it. Elsewhere in the Popularity component that I added the class names are necessary to add hover effects on table cells for example, which contain a Popularity component. It's necessary to target a specific div within the popularity to add a different color which takes into account the specific context. In this case, hovering a table row makes the background grey, so its necessary to make the non-filled in background of the popularity bar white, whereas if white was the default hover effect in the component it would do nothing on pages where the background is always white. The point here is that we don't highlight the popularity bar when its hovered, but when one if its parent's which is hovered, that parent being much larger in regards to dimensions in the browser.

Reply:
its hard to explain with words but it does this when you hover the table cell. 

![image](https://user-images.githubusercontent.com/24510397/120260250-855dcd00-c263-11eb-98c8-b11826d825d1.png)